core/commands/storage.go

```diff
@@ -455,7 +472,7 @@ func retryMonitor(ctx context.Context, api coreiface.CoreAPI, ss *storage.FileCo
 			}
 
 			// build connection with host, init step, could be error or timeout
-			go retryProcess(param, nil, true)
+			go retryProcess(param, nil, true, n, customizedSchedule, period)
```

Comment:
put the two new params inside `param` so we don't have to modify it further in the future